### DISCUSSION TITLE
Using Antimicrox with Xcloud?

### DISCUSSION THREAD

**Discussion by D1Ck3n** [AntiMicroX/antimicrox#295] [Q&A] [answered]
Hi all,

would it be possible to use the tool the other way around? For example, Xcloud from Microsoft has no keyboard + mouse support. Could it be fixed with this?

**Comment by pktiuk** [comment, marked as answer]
No, AntiMicroX maps gamepad to mouse + keyboard.  
If Xcloud doesn't support them, then AntiMicroX won't help.
